scripts/core/exp-command-buffer.yml

```
@@ -0,0 +1,300 @@
+name: "$x_exp_command_buffer_handle_t"
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Create a Command-Buffer object"
+class: $xCommandBuffer
+name: CreateExp
+decl: static
+details:
+    - "Create a command-buffer object"
+params:
+    - type: $x_context_handle_t
+      name: hContext
+      desc: "[in] handle of the context object"
+    - type: $x_device_handle_t
+      name: hDevice
+      desc: "[in] handle of the device object"
+    - type: "const $x_exp_command_buffer_desc_t*"
+      name: pCommandBufferDesc
+      desc: "[in][optional] CommandBuffer descriptor"
+    - type: "$x_exp_command_buffer_handle_t*"
+      name: phCommandBuffer
+      desc: "[out] pointer to Command-Buffer handle"
+returns:
+    - $X_RESULT_ERROR_INVALID_CONTEXT
+    - $X_RESULT_ERROR_INVALID_DEVICE
+    - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
+    - $X_RESULT_ERROR_OUT_OF_RESOURCES
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Increment the command-buffer object's reference count."
+class: $xCommandBuffer
+name: RetainExp
+params:
+    - type: $x_exp_command_buffer_handle_t
+      name: hCommandBuffer
+      desc: "[in] handle of the command-buffer object"
+returns:
+    - $X_RESULT_ERROR_INVALID_COMMAND_BUFFER_EXP
+    - $X_RESULT_ERROR_OUT_OF_RESOURCES
+    - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Decrement the command-buffer object's reference count and delete the command-buffer object if the reference count becomes zero."
+class: $xCommandBuffer
+name: ReleaseExp
+params:
+    - type: $x_exp_command_buffer_handle_t
+      name: hCommandBuffer
+      desc: "[in] handle of the command-buffer object"
+returns:
+    - $X_RESULT_ERROR_INVALID_COMMAND_BUFFER_EXP
+    - $X_RESULT_ERROR_OUT_OF_RESOURCES
+    - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Stop recording on a command-buffer object such that no more commands can be appended and make it ready to enqueue."
+class: $xCommandBuffer
+name: FinalizeExp
+params:
+    - type: $x_exp_command_buffer_handle_t
+      name: hCommandBuffer
+      desc: "[in] handle of the command-buffer object"
+returns:
+    - $X_RESULT_ERROR_INVALID_COMMAND_BUFFER_EXP
+    - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
+    - $X_RESULT_ERROR_OUT_OF_RESOURCES
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Append a kernel execution command to a command-buffer object"
+class: $xCommandBuffer
+name: AppendKernelExp
+params:
+    - type: $x_exp_command_buffer_handle_t
+      name: hCommandBuffer
+      desc: "[in] handle of the command-buffer object"
+    - type: $x_kernel_handle_t
+      name: hKernel
+      desc: "[in] kernel to append"
+    - type: uint32_t
+      name: WorkDim
```

```suggestion
      name: workDim
```

although I realised this may have been copied from [urEnqueueKernelLaunch](https://oneapi-src.github.io/unified-runtime/core/api.html#urenqueuekernellaunch) in which case I need to go canonicalise them following the contribution guide.

---

Fixed, thanks!